.golangci.yml

```diff
@@ -114,6 +114,7 @@ linters:
   # - goerr113
   # - maligned
   # - nestif
+  # - noctx (TODO: enable after next release; current release at time of writing is v1.27)
```

Comment:
Why can't it be enabled now?

Reply:
The golangci-lint version is 1.27  in Github Actions.
Version 1.27 does not include `noctx`, test in CI is failed.

https://github.com/golangci/golangci-lint/runs/745557992

```
 lint
    only-new-issues: false
Run golangci/golangci-lint-action@v1
  with:
    version: v1.27
    github-token: ***
    only-new-issues: false
prepare environment
go env
Installed Go in 15611ms
Prepared env in 15627ms

run golangci-lint
  Running [/home/runner/golangci-lint-1.27.0-linux-amd64/golangci-lint run --out-format=github-actions] in [] ...
  level=error msg="Running error: no such linter \"noctx\""
  
  ##[error]golangci-lint exit with code 3
  Ran golangci-lint in 125ms
```